exercises/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java

```diff
@@ -0,0 +1,214 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public final class LargestSeriesProductCalculatorTest {
+
+    @Rule
```

Comment:
Once discussion in #171 is complete, I will update with an appropriate comment to introduce the user to JUnit `Rule`s here.